pvnet/app.py

```diff
@@ -234,6 +248,9 @@ def app(
         .effective_capacity_mwp
     )
 
+    # National capacity is needed if using summation model
+    national_capacity = gsp_capacities.sum().item()
```

Comment:
should we take the sum here or should we take the national one from the database?

Reply:
I had thought about that, but actually when I ran the line `ds_gsp = next(iter(OpenGSPFromDatabase()))`, `ds_gsp` did not have any information for national - no`gsp_id=0`

Reply:
I've added a commit to pull the national capacity from the database